fcrepo-export/src/test/java/org/fcrepo/exporter/ExporterTest.java

```diff
@@ -146,6 +152,18 @@ public void testExportContainer() throws Exception {
         Assert.assertTrue(exporter.wroteFile(new File("target/rdf/rest/1.jsonld")));
     }
 
+    @Rule
+    public ExpectedException expectedEx = ExpectedException.none();
```

Comment:
Hang on, I just looked at this a bit more carefully. Actually, I think the right thing to do here is subclass `RuntimeException` with something specific to the condition (`AuthenticationRequiredException`). Then your test code is typesafe _and_ you can use the simpler `expected` form, right? I know that adds a little more code, but throwing naked `RuntimeException`s is always a bit suspect to me.